pkg/sbom/collectors/procfs/procfs_linux.go

```diff
@@ -0,0 +1,47 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+//go:build trivy
+
+package procfs
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/shirou/gopsutil/v4/process"
+
+	"github.com/DataDog/datadog-agent/pkg/sbom"
+	"github.com/DataDog/datadog-agent/pkg/security/secl/containerutils"
+	"github.com/DataDog/datadog-agent/pkg/security/utils"
```

Comment:
It's really not amazing that we start using some pkg/security stuff here, since it's going to be in the core agent

Reply:
yeah I agree, I'll see if I extract only what is needed or move it to a common package

Reply:
I removed this dependency. The algo is more naive but still work